File: app/src/main/java/com/example/concours_passerelle/HomeActivity.java
package com.example.concours_passerelle;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import com.example.concours_passerelle.Candidate.CreateProfileCandidateActivity;

public class HomeActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        // Initialiser les boutons
        Button btnCandidate = findViewById(R.id.btnCandidate);
        Button btnLogin = findViewById(R.id.btnLogin);

        // Configurer l'action au clic sur "Candidate"
        btnCandidate.setOnClickListener(v -> {
        // Démarrer CreateProfileCandidateActivity
        startActivity(new Intent(HomeActivity.this, CreateProfileCandidateActivity.class));
    });

        // Configurer l'action au clic sur "Login"
        btnLogin.setOnClickListener(v -> {
        // Démarrer LoginActivity
        startActivity(new Intent(HomeActivity.this, LoginActivity.class));
    });
    }
}

File: app/src/main/java/com/example/concours_passerelle/HomeActivity.kt
package com.example.concours_passerelle

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class HomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)

        val btnCandidate: Button = findViewById(R.id.btnCandidate)
        val btnLogin: Button = findViewById(R.id.btnLogin)

        btnCandidate.setOnClickListener {
            startActivity(Intent(this, CreateProfileActivity::class.java))
        }

        btnLogin.setOnClickListener {
            startActivity(Intent(this, LoginActivity::class.java))
        }
    }
}
